backend/streams_explorer/core/services/dataflow_graph.py

```diff
@@ -22,13 +26,23 @@ class NodeNotFound(Exception):
     pass
 
 
+class NodeDataFields(str, Enum):
+    NODE_TYPE = "node_type"
+    LABEL = "label"
+
+
 class DataFlowGraph:
-    def __init__(self, metric_provider: Type[MetricProvider]):
+    def __init__(self, metric_provider: Type[MetricProvider], kafka: KafkaAdminClient):
         self.graph = nx.DiGraph()
         self.json_graph: dict = {}
         self.pipelines: Dict[str, nx.DiGraph] = {}
         self.json_pipelines: Dict[str, dict] = {}
         self.metric_provider_class = metric_provider
+        self.kafka = kafka
+
+        self._topic_pattern_queue: Dict[str, Set[str]] = defaultdict(
+            set
+        )  # topic pattern -> list of target node ids
```

Comment:
sorry about nitpick :D
```suggestion
        )  # topic pattern -> set of target node ids
```

Reply:
Never be sorry for being nitpicky in reviews ;-)